packages/amplify-e2e-tests/src/__tests__/function_11.test.ts

```diff
@@ -0,0 +1,175 @@
+import {
+  addFunction,
+  amplifyPush,
+  createNewProjectDir,
+  deleteProject,
+  deleteProjectDir,
+  getBackendConfig,
+  initJSProjectWithProfile,
+  getProjectMeta,
+  getFunction,
+  functionBuild,
+  invokeFunction,
+  updateApiSchema,
+  generateRandomShortId,
+  addApi,
+
+} from '@aws-amplify/amplify-e2e-core';
+
+describe('Lambda AppSync nodejs:', () => {
+  let projRoot: string;
+
+  beforeEach(async () => {
+    projRoot = await createNewProjectDir('lambda-appsync-nodejs');
+  });
+
+  afterEach(async () => {
+    await deleteProject(projRoot);
+    deleteProjectDir(projRoot);
+  });
+
+  it('API not present', async () => {
```

Comment:
Could you provide slightly more describe names for these 'setup' style tests? Just something like it('deploys when API is not present'), and then we probably want to verify the state of the system that we added an API or something like that.

Reply:
Thank you, modified to provide better descriptive names for the tests